src/support.js

```diff
@@ -126,15 +127,20 @@ function processImage (src, longOptions) {
   }
 
   var shortOptions = Object.assign({}, DEFAULT_OPTIONS)
-  Object.keys(longOptions).forEach(function (longKey) {
-    var value = longOptions[longKey]
-    var shortKey = PARAM_EXPANSION[longKey]
-    if (shortKey) {
-      shortOptions[shortKey] = value
-    } else {
-      // fall back for any values that aren't in expansion defined above
-      shortOptions[longKey] = value
+  Object.keys(longOptions).forEach(function (key) {
+    var val = longOptions[key]
+
+    if (PARAM_EXPANSION[key]) {
+      key = PARAM_EXPANSION[key]
     }
+
+    key = encodeURIComponent(key)
+
+    if (key.substr(-2) === '64') {
```

Comment:
While `encodeURIComponent` is probably a good idea to add, `js-base64` (a 6.6kb dep) and these next few lines seem very user-landy. As in it doesn't seem to be the most general case that most or many users of this library will get use from.